packages/teleport/src/Audit/Audit.story.tsx

```diff
@@ -28,11 +29,22 @@ export default {
 export const Loaded = () => {
   const ctx = new Context();
   ctx.auditService.fetchEvents = () =>
-    Promise.resolve({ events, startKey: '' });
+    Promise.resolve({ events: events.slice(0, 20), startKey: '' });
 
   return render(ctx);
 };
 
+export const AllEventsList = () => (
```

Comment:
```suggestion
export const AllEvents = () => (
```

and lets place this at the bottom of file, so its last in storybook list